backend/src/game/gateway/game.gateway.ts

```diff
@@ -54,21 +58,34 @@ export class GameGateway implements OnGatewayInit, OnGatewayConnection, OnGatewa
 		}
 		player.gameSocket = client;
 		client.data.userId = userId;
+		this.userService.status(userId, UserStatus.IN_GAME);
```

Comment:
Change to 'entry game' event on this file

Reply:
ja esta a ser feito no proximo merge nao e necessario agora